src/presign/participant.rs

```diff
@@ -1053,7 +1054,10 @@ impl PresignKeyShareAndInfo {
         )?;
 
         Ok((
-            round_two::Private { beta, beta_hat },
+            round_two::Private {
+                beta: beta.into(),
```

Comment:
Analogous to my previous comment. `beta` should probably be instantiated as a `SecretBigNumber`, where it is generated. And all code that uses it should use `SecretBigNumber` or a reference: `&BigNumber`